api/v0/cells/cells.proto

```diff
@@ -225,3 +221,39 @@ message CpusetController {
 
   // cpus_partition is not supported
 }
+
+// Docs:
+// https://docs.kernel.org/admin-guide/cgroup-v2.html#memory-interface-files
+message MemoryController {
+  // Hard memory protection. If the memory usage of a cgroup is within its
+  // effective min boundary, the cgroup’s memory won’t be reclaimed under any
+  // conditions. If there is no unprotected reclaimable memory available, OOM
+  // killer is invoked. Above the effective min boundary (or effective low
+  // boundary if it is higher), pages are reclaimed proportionally to the
+  // overage, reducing reclaim pressure for smaller overages.
+  // NOTE: unused by aurae
+  optional int64 min = 1;
+
+  // Best-effort memory protection. If the memory usage of a cgroup is within
+  // its effective low boundary, the cgroup’s memory won’t be reclaimed unless
+  // there is no reclaimable memory available in unprotected cgroups. Above
+  // the effective low boundary (or effective min boundary if it is higher),
+  // pages are reclaimed proportionally to the overage, reducing reclaim
+  // pressure for smaller overages.
+  // NOTE: unused by aurae
```

Comment:
Low is used. High is not

Reply:
some documentation may be out of date. :)